packages/atomic/src/components/result-lists/result-list.tsx

```diff
@@ -0,0 +1,120 @@
+import {FunctionalComponent, h, Host} from '@stencil/core';
+import {updateBreakpoints} from '../../utils/replace-breakpoint';
+import {once} from '../../utils/utils';
+import {
+  getResultDisplayClasses,
+  ResultDisplayLayout,
+} from '../atomic-result/atomic-result-display-options';
+import {
+  AtomicResultListBase,
+  ResultPlaceholderProps,
+  ResultsProps,
+} from './result-list-common';
+import {TableDisplayResultsPlaceholder} from './table-display-results-placeholder';
+import {TableDisplayResults} from './table-display-results';
+import {ListDisplayResults} from './list-display-results';
+import {GridDisplayResults} from './grid-display-results';
+import {GridDisplayResultsPlaceholder} from './grid-display-results-placeholder';
+import {ListDisplayResultsPlaceholder} from './list-display-results-placeholder';
+interface ResultListProps {
```

Comment:
would probably rename this `BaseResultListProps` and export it

Reply:
why export it? I don't think it's being used anywhere else